app/models/user.rb

```diff
@@ -111,6 +111,12 @@ def invitation_status
     "invited" if invitation_sent_at.present?
   end
 
+  def mark_invitation_status_as_accepted!
+    self.invitation_sent_at = Time.current
```

Comment:
❓Question: Should I be concerned about the other related invitation attributes? Like, for example, the attributes `invitation_created_at`, or `invitation_limit`, or `invitation_token`, etc…

Reply:
Nope.